scss/multiselect/_layout.scss

```diff
@@ -9,6 +9,11 @@
             visibility: hidden;
         }
 
+        .k-clear-value {
+            top: $padding-x-lg / 2;
```

Comment:
The value fixes the issue, consider using a `y`-unit when calculating vertical offset. If left like this, changing the `padding-x-lg` will offset the icon vertically, which is not intuitive for the developers who use the theme.